grid_map_pcl/src/grid_map_pcl_loader_node.cpp

```diff
@@ -15,34 +20,32 @@
 
 namespace gm = ::grid_map::grid_map_pcl;
 
-int main(int argc, char** argv) {
-  ros::init(argc, argv, "grid_map_pcl_loader_node");
-  ros::NodeHandle nh("~");
-  gm::setVerbosityLevelToDebugIfFlagSet(nh);
+int main(int argc, char ** argv)
+{
+  rclcpp::init(argc, argv);
+  auto node = std::make_shared<rclcpp::Node>("grid_map_pcl_loader_node");
+  gm::setVerbosityLevelToDebugIfFlagSet(node);
 
-  ros::Publisher gridMapPub;
-  gridMapPub = nh.advertise<grid_map_msgs::GridMap>("grid_map_from_raw_pointcloud", 1, true);
+  auto gridMapPub = node->create_publisher<grid_map_msgs::msg::GridMap>(
+    "grid_map_from_raw_pointcloud", rclcpp::QoS(1).transient_local());
```

Comment:
No action required, `rclcpp::QoS(1).transient_local()` I think in terms of readability this is a downgrade

Reply:
Moved the setting of the QoS to a variable, hope that helps a bit.